rpc_jobs/irr_role_test.yml

```diff
@@ -80,14 +80,17 @@
     image:
       - trusty:
           IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+          irr_flavor: "performance2-15"
     # Context is used to define the origin "release" when leap upgrading
     #  The "series" parameter is used to define the checkout within the
     #  release.
     context:
+      - kilo
       - kilo:
```

Comment:
If you use the context of ``r11.0.0`` or ``kilo_r11.0.0`` then the series does not need to change in the job definition.